Automattic/jetpack
# Untangle: Users > My Profile to default wp-admin Users > Profile

Fixes https://github.com/Automattic/dotcom-forge/issues/5258

## Proposed changes:
Part of Untangle Calypso.
Remove Users > My profile in favor of default Users > Profile for users with Classic interface
Removed the second Language field and make the Language selector works.

| Before | After |
| --- | --- |
| <img width="557" alt="image" src="https://github.com/Automattic/jetpack/assets/402286/659dadc2-fc9c-4a39-9e1b-62f50c12fca9"> | <img width="548" alt="image" src="https://github.com/Automattic/jetpack/assets/402286/9fd8cf54-2828-46b5-92b4-38d93ee89828"> |

## Jetpack product discussion
paYJgx-4uN-p2#comment-4524

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Set your `Admin interface style` to `Classic (wp-admin)` in `wordpress.com/hosting-config/site_slug`
* Go to `wp-admin`
* `Users` > `Profile` should link to wp-admin profile page
